docs/operating-scylla/index.rst

```diff
@@ -10,6 +10,7 @@ Scylla for Administrators
    security/index
    admin-tools/index
    manager/index
+   ScyllaDB Monitoring Stack <https://monitoring.docs.scylladb.com/stable/>
```

Comment:
Its better to use <https://monitoring.docs.scylladb.com> without the "stable"

Reply:
@tzach Corrected.